docs/index.asciidoc

```diff
@@ -337,3 +337,30 @@ NOTE: Client identity is not typically validated using SSL because the receiving
 include::{include_path}/{type}.asciidoc[]
 
 :default_codec!:
+
+==== Elasticsearch output
```

Comment:
Please relocate this helpful info to around Line 134.  It's getting lost behind the Configuration and Common options. 

`:default_codec!:` should be the last line in the file.